ccm_web/package.json

```diff
@@ -5,7 +5,8 @@
   "scripts": {
     "dev:server": "nodemon --inspect=0.0.0.0:9229 --config nmon_config.json server/src/main.ts",
     "dev:client": "webpack --watch --config webpack/webpack.dev.ts",
-    "dev": "npm run dev:server & npm run dev:client",
+    "predev": "wait-port ccm_db:3306",
+    "dev": "node -r ts-node/register server/migrator up && npm run dev:server & npm run dev:client",
```

Comment:
I don't think it will work. here is the logs

```
(i46_orm_db)$ docker exec -it ccm_web ts-node server/migrator up
OCI runtime exec failed: exec failed: container_linux.go:367: starting container process caused: exec: "ts-node": executable file not found in $PATH: unknown

(i46_orm_db)$ docker exec -it ccm_web ts-node/register server/migrator up
OCI runtime exec failed: exec failed: container_linux.go:367: starting container process caused: exec: "ts-node/register": stat ts-node/register: no such file or directory: unknown

Working case
(i46_orm_db)$ docker exec -it ccm_web node -r ts-node/register server/migrator up

2021-05-20T13:41:07.919Z - config.ts - info - Value undefined for LTI_ENCRYPTION_KEY is invalid; using fallback LTIKEY
2021-05-20T13:41:08.049Z - migrator.ts - info - Running migrations CLI...
Executing (default): CREATE TABLE IF NOT EXISTS `SequelizeMeta` (`name` VARCHAR(255) NOT NULL UNIQUE , PRIMARY KEY (`name`)) ENGINE=InnoDB DEFAULT CHARSET=utf8 COLLATE utf8_unicode_ci;
Executing (default): SHOW INDEX FROM `SequelizeMeta`
Executing (default): SELECT `name` FROM `SequelizeMeta` AS `SequelizeMeta` ORDER BY `SequelizeMeta`.`name` ASC;
2021-05-20T13:41:08.228Z - migrator.ts - info - applied 0 migrations.
2021-05-20T13:41:08.228Z - migrator.ts - info - Migration tasks ran!
```

Reply:
Hm, okay.